packages/cli/tsconfig.json

```diff
@@ -1,12 +1,21 @@
 {
-  "extends": "../../tsconfig.json",
+  "extends": "../../tsconfig.base.json",
   "compilerOptions": {
     "module": "commonjs",
+    "declaration": false,
+    "declarationMap": false,
+    "noImplicitAny": false,
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
❓ **question:** ‏ Is there a good reason for setting `noImplicitAny` to false?

Reply:
No reason at all. Great catch.